api.planx.uk/modules/analytics/metabase/collection/types.ts

```diff
@@ -0,0 +1,54 @@
+import { z } from "zod";
+import type { ValidatedRequestHandler } from "../../../../shared/middleware/validate.js";
+
+type ApiResponse<T> = {
+  data?: T;
+  error?: string;
+};
+
+export const checkCollectionsSchema = z.object({
+  query: z.object({
+    name: z.string().min(1, "Name parameter is required"),
+  }),
+});
+
+export type CheckCollectionsRequest = ValidatedRequestHandler<
+  typeof checkCollectionsSchema,
+  ApiResponse<CheckCollectionResponse>
+>;
+
+export interface CheckCollectionResponse {
+  collectionId: number | false;
+}
+
+export interface NewCollectionParams {
+  /** The name of the new collection */
+  name: string;
+  description?: string;
+  /** Optional; if the collection is a child of a parent, specify parent ID here
+   * For council teams, parent collection should be 58
+   */
+  parent_id?: number;
+  namespace?: string;
+  authority_level?: null;
+}
+
+export const newCollectionSchema = z.object({
+  body: z.object({
+    name: z.string(),
+    description: z.string().optional(),
+    parent_id: z.number().optional(),
```

Comment:
I think it's safe to assume all new collections will be grouped here by default.

```ts
/** Metbase collection ID for the the "Council" collection **/
const COUNCIL_COLLECTION_ID = 58;

...
parent_id: z.number().default(COUNCIL_COLLECTION_ID)
```